www/app.py

```diff
@@ -0,0 +1,27 @@
+from flask import Flask, jsonify, Response
+from os import listdir
+from os.path import isfile, join
+from flask_cors import CORS, cross_origin
```

Comment:
Not yet! Want to add one? (I'm curious to make sure you have the permissions to do it)